Makefile

```diff
@@ -119,6 +119,7 @@ BUILD_ARCHITECTURE ?= amd64
 
 DOCKER_BUILD_FLAGS ?= --platform linux/$(BUILD_ARCHITECTURE)
 DOCKER_RUN_FLAGS ?= --add-host=host.docker.internal:host-gateway --rm -it
+DOCKER_DEBUG_FLAGS ?= -it --entrypoint="bash" -v /tmp/.X11-unix:/tmp/.X11-unix -e DISPLAY=$$DISPLAY
```

Comment:
It would be great to add a short explanation to the PR description and/or commit message as to why the `-v /tmp/.X11-unix:/tmp/.X11-unix` was introduced here.

Reply:
Done.